plugins/modules/win_domain_computer.ps1

```diff
@@ -146,6 +157,73 @@ Function Add-ConstructedState($desired_state) {
   $result.changed = $true
 }
 
+Function Invoke-OfflineDomainJoin {
+  [CmdletBinding(SupportsShouldProcess=$true)]
+  param(
+    [Parameter(Mandatory=$true)]
+    [System.Collections.IDictionary]
+    $desired_state ,
+
+    [Parameter()]
+    [bool]
+    $ReturnBlob = $false ,
+
+    [Parameter()]
+    [System.IO.FileInfo]
+    $BlobPath
+  )
+
+  End {
```

Comment:
If you don't have a being/process/end block just omit it entirely. The default in the absence of these blocks is `End {}`

Reply:
Like `[Parameter()]` it's my own style to make it explicit, but I can remove if you prefer.